src/usr/local/www/status_openvpn.php

```diff
@@ -351,84 +313,67 @@ function(index,row) { jQuery(row).fadeOut(1000); }
+	<div class="panel-heading"><h2 class="panel-title"><?=gettext("Client Instance Statistics"); ?></h2></div>
+		<div class="panel-body table-responsive">
+			<table class="table table-striped table-hover">
+				<thead>
+					<tr>  
+						<th><?=gettext("Name"); ?></th>
+						<th><?=gettext("Status"); ?></th>
+						<th><?=gettext("Connected Since"); ?></th>
+						<th><?=gettext("Virtual Addr"); ?></th>
+						<th><?=gettext("Remote Host"); ?></th>
+						<th><?=gettext("Bytes Sent"); ?></th>
+						<th><?=gettext("Bytes Rcvd"); ?></th>
+						<th><?=gettext("Service"); ?></th>
+					</tr>
+				</thead>
+				<tbody>
 
 <?php
 		foreach ($clients as $client):
 ?>
-			<tr id="<?php echo "r:{$client['port']}:{$client['vpnid']}"; ?>">
-				<td class="listlr">
-					<?=htmlspecialchars($client['name']);?>
-				</td>
-				<td class="listr">
-					<?=$client['status'];?>
-				</td>
-				<td class="listr">
-					<?=$client['connect_time'];?>
-				</td>
-				<td class="listr">
-					<?=$client['virtual_addr'];?>
-				</td>
-				<td class="listr">
-					<?=$client['remote_host'];?>
-				</td>
-				<td class="listr">
-					<?=format_bytes($client['bytes_sent']);?>
-				</td>
-				<td class="listr">
-					<?=format_bytes($client['bytes_recv']);?>
-				</td>
-				<td class="listr" height="12">
-				<table>
-				<tr>
-								<td>
-									<?php $ssvc = find_service_by_openvpn_vpnid($client['vpnid']); ?>
-				<?= get_service_status_icon($ssvc, false, true); ?>
-									<?= get_service_control_links($ssvc, true); ?>
-								</td>
-				</tr>
-				</table>
-				</td>
-			</tr>
+					<tr id="<?php echo "r:{$client['port']}:{$client['vpnid']}"; ?>">
+						<td><?=htmlspecialchars($client['name']);?></td>
+						<td><?=$client['status'];?></td>
+						<td><?=$client['connect_time'];?></td>
+						<td><?=$client['virtual_addr'];?></td>
+						<td><?=$client['remote_host'];?></td>
+						<td><?=format_bytes($client['bytes_sent']);?></td>
+						<td><?=format_bytes($client['bytes_recv']);?></td>
+						<td>
+							<table>
+								<tr>
+									<td>
+										<?php $ssvc = find_service_by_openvpn_vpnid($client['vpnid']); ?>
+										<?= get_service_status_icon($ssvc, false, true); ?>
+										<?= get_service_control_links($ssvc, true); ?>
+									</td>
+								</tr>
+							</table>
+						</td>
+					</tr>
 <?php
 		endforeach;
 ?>
+				</tbody>
 			</table>
-		</td>
-	</tr>
-</table>
+		</div>
+</div>
 
 <?php 
 }
 
 if ($DisplayNote) {
-	echo "<br /><b>" . gettext("NOTE") . ":</b> " . gettext("If you have custom options that override the management features of OpenVPN on a client or server, they will cause that OpenVPN instance to not work correctly with this status page.");
+ 	print_info_box("<br /><b>" . gettext("NOTE") . ":</b> " . gettext("If you have custom options that override the management features of OpenVPN on a client or server, they will cause that OpenVPN instance to not work correctly with this status page."));
 }
```

Comment:
I have been removing the word "Note" when using print_info_box. info boxes are notes by definition so it is not needed. I have also removed unneeded phrases such as "Click here to . ." and "Options entered here . . "